### DISCUSSION TITLE
mspider/user-guide/service-list/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1195] [Announcements]
# mspider/user-guide/service-list/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-list/

**Comment by liyuerich** [comment]
在服务列表右侧，点击 ⋮ 选择相应的菜单项，可以跳转至流量治理和安全治理。

新的版本中菜单项已经不一样了

**Comment by liyuerich** [comment]
能提供网格概览页面的描述吗？

**Comment by liyuerich** [comment]
请特别关注服务列表中诊断配置这一列。当诊断信息为异常时，光标悬浮在 ⓘ 上将显示异常原因。异常状态会影响下一阶段的服务流量治理等网格相关能力。

异常 能在哪里解决呢？
